resources/documentation/deploy_bluebanquise.rst

```diff
@@ -148,6 +148,18 @@ Note also that if you update configuration on management1, it is recommanded to
 
 Please refer to the pxe_stack role dedicated section in this documentation for more information on the bootset usage.
 
+SSH public key
+--------------
+
+In order to log into the remote nodes without giving the password, check that
+the ssh public key defined in authentication.yml in your inventory match your
+management1 public key. If not, update it. Remember to run the pxe_stack role
+after updating the configuration.
+
+.. code-block:: bash
+
+  bluebanquise-playbook -p management1 -n management1 --tags pxe_stack
```

Comment:
I would prefer to keep ansible playbook command in main documentation, as bluebanquise-playbook is an additional tool, for production usage designed to facilitate usage.

Reply:
> every one used to Ansible can understand the whole stack quickly :-)

The project has some specific that don't follow Ansible guidelines, it's not that easy to understand. :-)

Anyway, your comment makes sense, I will update the PR.